tcf_website/tests/test_commands.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for Django management commands"""
+from django.core import management
+from django.test import TestCase
+
+from tcf_website.models import CourseGrade, CourseInstructorGrade
+from .test_utils import setup
+
+
+class LoadGradesTestCase(TestCase):
+    """ Tests for the load_grades command. Uses the test_data.csv file, which holds dummy
+    data for two sections of the same course taught by the same instructor."""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        setup(cls)
+        management.call_command('load_grades', 'test_data', '--suppress-tqdm', verbosity=0)
+        cls.cg = CourseGrade.objects.all()[0]
+        cls.cig = CourseInstructorGrade.objects.all()[0]
+
+    def test_no_duplicates(self):
+        """Make sure only one instance of CourseGrade and CourseInstructorGrade were created"""
+        self.assertEqual(len(CourseGrade.objects.all()), 1)
+        self.assertEqual(len(CourseInstructorGrade.objects.all()), 1)
+
+    def test_correct_course(self):
+        """Make sure the course for both is CS 420"""
+        self.assertEqual(self.cg.course, self.cig.course)
+
+    def test_correct_instructor(self):
+        """Make sure instructor is Tom Jefferson"""
+        self.assertEqual(self.cig.instructor.full_name(), "Tom Jefferson")
+
+    def test_total_students(self):
+        """Check valid total_enrolled"""
+        self.assertEqual(self.cg.total_enrolled, self.cig.total_enrolled)
+        self.assertEqual(self.cg.total_enrolled, 24)
```

Comment:
```suggestion
        self.assertEqual(self.cg.total_enrolled, 24)
        self.assertEqual(self.cig.total_enrolled, 24)
```
I think this would be easier to debug.